### ISSUE TITLE
symprec sensitive segfault for get_symmetry

### ISSUE THREAD

**Issue by blondegeek** [atztogo/spglib#19]
Hello,

I'm finding that for some structures get_symmetry will result with a segfault and that the segfault is symprec sensitive. Please see the examples below.

Thanks!
Tess

Example 1:

input
```
import spglib
cell = ([[  3.94000000e+00,   0.00000000e+00,   2.41255419e-16],
         [ -2.41255419e-16,   3.94000000e+00,   2.41255419e-16],
         [  0.00000000e+00,   0.00000000e+00,   3.94000000e+00]],
        [[ 0.971,  0.971,  0.971],
         [ 0.971,  0.029,  0.971],
         [ 0.971,  0.971,  0.029],
         [ 0.029,  0.971,  0.971],
         [ 0.029,  0.971,  0.029],
         [ 0.029,  0.029,  0.029],
         [ 0.971,  0.029,  0.029],
         [ 0.029,  0.029,  0.971],
         [ 0.   ,  0.   ,  0.   ],
         [ 0.95 ,  0.95 ,  0.5  ],
         [ 0.05 ,  0.95 ,  0.5  ],
         [ 0.   ,  0.   ,  0.5  ],
         [ 0.05 ,  0.05 ,  0.5  ],
         [ 0.95 ,  0.05 ,  0.5  ],
         [ 0.95 ,  0.5  ,  0.95 ],
         [ 0.   ,  0.5  ,  0.   ],
         [ 0.05 ,  0.5  ,  0.95 ],
         [ 0.95 ,  0.5  ,  0.05 ],
         [ 0.05 ,  0.5  ,  0.05 ],
         [ 0.5  ,  0.95 ,  0.95 ],
         [ 0.5  ,  0.05 ,  0.05 ],
         [ 0.5  ,  0.   ,  0.   ],
         [ 0.5  ,  0.05 ,  0.95 ],
         [ 0.5  ,  0.95 ,  0.05 ]],
        [1, 1, 1, 1, 1, 1, 1, 1, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3],
        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])

spglib.get_symmetry(cell,symprec=0.5)
print("symprec=0.5 ok")
spglib.get_symmetry(cell,symprec=0.6)
print("symprec=0.6 ok")
```

output
```
symprec=0.5 ok
Segmentation fault: 11
```

Example structure 2:

input
```
import spglib 

cell = ([[  5.40600000e+00,   0.00000000e+00,   3.31022030e-16],
         [ -3.28817666e-16,   5.37000000e+00,   3.28817666e-16],
         [  0.00000000e+00,   0.00000000e+00,   1.31500000e+01]],
        [[ 0.493,  0.   ,  0.138],
         [ 0.507,  0.   ,  0.138],
         [ 0.993,  0.5  ,  0.862],
         [ 0.007,  0.5  ,  0.862],
         [ 0.993,  0.5  ,  0.138],
         [ 0.007,  0.5  ,  0.138],
         [ 0.493,  0.   ,  0.862],
         [ 0.507,  0.   ,  0.862],
         [ 0.993,  0.   ,  0.638],
         [ 0.007,  0.   ,  0.638],
         [ 0.493,  0.5  ,  0.638],
         [ 0.507,  0.5  ,  0.638],
         [ 0.007,  0.   ,  0.362],
         [ 0.993,  0.   ,  0.362],
         [ 0.507,  0.5  ,  0.362],
         [ 0.493,  0.5  ,  0.362],
         [ 0.5  ,  0.   ,  0.5  ],
         [ 0.   ,  0.5  ,  0.5  ],
         [ 0.   ,  0.   ,  0.   ],
         [ 0.5  ,  0.5  ,  0.   ],
         [ 0.531,  0.   ,  0.313],
         [ 0.469,  0.   ,  0.313],
         [ 0.25 ,  0.25 ,  0.507],
         [ 0.25 ,  0.25 ,  0.493],
         [ 0.25 ,  0.75 ,  0.493],
         [ 0.25 ,  0.75 ,  0.507],
         [ 0.75 ,  0.25 ,  0.507],
         [ 0.75 ,  0.25 ,  0.493],
         [ 0.75 ,  0.75 ,  0.493],
         [ 0.75 ,  0.75 ,  0.507],
         [ 0.031,  0.5  ,  0.687],
         [ 0.969,  0.5  ,  0.687],
         [ 0.031,  0.5  ,  0.313],
         [ 0.969,  0.5  ,  0.313],
         [ 0.531,  0.   ,  0.687],
         [ 0.469,  0.   ,  0.687],
         [ 0.031,  0.   ,  0.813],
         [ 0.969,  0.   ,  0.813],
         [ 0.531,  0.5  ,  0.813],
         [ 0.469,  0.5  ,  0.813],
         [ 0.969,  0.   ,  0.187],
         [ 0.031,  0.   ,  0.187],
         [ 0.75 ,  0.75 ,  0.993],
         [ 0.75 ,  0.75 ,  0.007],
         [ 0.25 ,  0.25 ,  0.007],
         [ 0.25 ,  0.25 ,  0.993],
         [ 0.25 ,  0.75 ,  0.993],
         [ 0.25 ,  0.75 ,  0.007],
         [ 0.75 ,  0.25 ,  0.007],
         [ 0.75 ,  0.25 ,  0.993],
         [ 0.469,  0.5  ,  0.187],
         [ 0.531,  0.5  ,  0.187]],
        [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3],
        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])

spglib.get_symmetry(cell,symprec=0.5)
print("symprec=0.5 ok")
spglib.get_symmetry(cell,symprec=0.6)
print("symprec=0.6 ok")
spglib.get_symmetry(cell,symprec=0.7)
print("symprec=0.7 ok")
```

output
```
symprec=0.5 ok
symprec=0.6 ok
Segmentation fault: 11
```

**Comment by atztogo** [comment]
Hello,

Thanks for reporting the segfaults. Just a few days ago, this was fixed around 261692756d986471afd2356a7c76b961084eb3d3  and now it is in develop branch. The reason of the segfaults is that some distance of atoms is too close. I hope this version will be soon released. Please wait for a while...

Togo

**Comment by blondegeek** [comment]
Great! Thank you!
